app/utils/utils.tsx

```diff
@@ -34,3 +35,19 @@ export const countdownRenderer = ({ hours, minutes, seconds }) => (
     {zeroPad(hours)}:{zeroPad(minutes)}:{zeroPad(seconds)}
   </span>
 );
+
+// We re-export useHotkeys with preventDefault on the handler
```

Comment:
I added left, right and space as hotkeys – these have default behaviors that shift the viewport on the page.
These defaults are not really useful in Chessguessr, where the entire UI fits horizontally on anything that's got a keypad, and no inputs where a space is helpful (arrow-down can be used to move viewport down).
In order not to clutter all the `useHotkey` calls with extra code to prevent the default, I just made an override in the utils